apps/challenges/views.py

```diff
@@ -1345,6 +1345,9 @@ def get_broker_urls(request):
     else:
         challenges = Challenge.objects.filter(**q_params)
         response_data = challenges.values_list('queue', flat=True)
+        response_data = {
+                'result': list(response_data)
```

Comment:
I think we can just return the `{'results': response_data}` instead of converting it to the list. Also, please note that the key is `results` and not `result`.

Reply:
Done!